src/OrdersList/Orders.cpp

```diff
@@ -76,19 +93,23 @@ Deploy::Deploy(Deploy *d) {
     this->addDescription();
 }
 
+/**
+* Checks player state and executes order
+* @param currentState player's current state
+*/
 void Deploy::execute(State* current) {
     if (current->getStateName().compare("executeorders")==0 && validate() == 1){
         cout << "Executing (Deploy) order #" << getOrderID() << " ..." << endl;
     } else cout << "Can NOT execute (Deploy) order #" << getOrderID() << " ..." << endl;
 };
 
 void Deploy::addDescription() {
-    this->description = "(Deploy) Move a certain number of army units from the current player's \n"
+    *description = "(Deploy) Move a certain number of army units from the current player's \n"
         "            reinforcement pool to one of the current player's territories.";
 }
 
 string Deploy::getDescription() {
-    return this->description;
+    return *description;
 }
 
 /******************************* Advance *********************************************/
```

Comment:
minor!!!!
for all order types, my OCD screams that the number of stars isnt equalllll!!
use this template:
`/************************************************************ Player **************************************************************/`